app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,94 @@
+<div class="ui hidden divider"></div>
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
```

Comment:
it will be handled by PR [#144](https://github.com/fossasia/open-event-frontend/pull/144/files)